server/api/region.go

```diff
@@ -64,7 +64,7 @@ func (h *regionHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	region, leader := cluster.GetRegionAndLeader(regionID)
+	region, leader := cluster.GetRegionByID(regionID)
```

Comment:
do we need using a structure to hold region and leader?
If we want to return another info, how do we do?